_posts/en/newsletters/2018-07-10-newsletter.md

```diff
@@ -147,20 +124,19 @@ to pursue later." ([source][pwuille comment])
     the information you display to your users.  Those nodes will still
     see all confirmed transactions in valid blocks.
 
-    Note that to lower the minimum relay fee in Bitcoin Core below its
-    default, you need to change two settings.  Shown below are the two
-    settings with their default values in Bitcoin Core 0.16.1; to lower
-    the values, change both of them to the same value, but be aware that
-    reducing them too far (perhaps to less than 1/10th the default)
+    To lower the minimum relay fee in Bitcoin Core below its
+    default, you need to change a config setting.  Shown below is the
+    setting with its default value in Bitcoin Core 0.16.1; to lower
+    the value, restart bitcoind with the following config, but be aware that
+    reducing it too far (perhaps to less than 1/10th the default)
     exposes you to bandwidth-wasting attacks and reduces BIP152 compact
     block efficiency for your node.
 
       minrelaytxfee=0.00001000
-      incrementalrelayfee=0.00001000
```

Comment:
I think this is wrong, the actual minimum feerate is the greater of those two settings. Consider the return for GetMinFee():

    return std::max(CFeeRate(llround(rollingMinimumFeeRate)), incrementalRelayFee);

Also, the reason I think this is wrong is because I tried setting minrelaytxfee below the minimum on my node a few months ago and, after a day, I observed that I had no below-default-fee transactions. So I poked around and added incrementalrelayfee with the same setting, and within minutes started to receive below default-min transactions. Feel free to try the same experiment yourself.

Reply:
hmmm, yes it looks like you're right, but I can't understand why that code is like that. I think it effectively makes incrementalrelayfee a floor for the minrelaytxfee.

It was changed here: https://github.com/bitcoin/bitcoin/pull/9380/commits/7b1add3c28aae8caf2e1517f15cd953eacbb4931#diff-ca74c4b28865382863b8fe7633a85cd6R1084

I'm going to investigate whether that change was intentional. For now, I'll revert the newsletter to your version since that's the current behaviour.